README.md

```diff
@@ -11,6 +11,25 @@ Steps before running Open school backend on Windows:
 * Set property ```-Dspring.profiles.active=local``` in Intellij Idea Run/Debug Configuration VM Options\
   If you run via Gradle-based Run Configuration, add `SPRING_PROFILES_ACTIVE=local` to Intellij Idea Run/Debug Configuration Environment Variables instead.
 
+## Running locally without Docker (using H2 database)
+
+Steps before running Open school project on Windows:
+
+* Clone the project (if the project already exists on the computer, skip this step)
```

Comment:
as we have the docker configuration step above, could you please add the option to run the application with a 'local' profile?